featuresScore.java

```diff
@@ -0,0 +1,76 @@
+
+public class featuresScore {
+	public static final int COLS = 10;
+	public static final int ROWS = 21;
```

Comment:
I took the constant from the State class file. The documentation said that the h = 20 though. So I am confused....